### DISCUSSION TITLE
Poetry support for package management

### DISCUSSION THREAD

**Discussion by tonytziorvas** [drivendataorg/cookiecutter-data-science#375] [Requested tools]
Although the folder structure is amazing and pretty up-to-date with the current standards, I believe there are better alternatives for package management than pip. Poetry has been in the field for long enough and is proven to be better at managing dependencies and grouping them.
I would really love to see this template utilize Poetry or at least give an option to use poetry instead of pip when downloading the template. (duplicate structure with different dependancy maybe... ?) 

Here is a template I found that does that. 
https://khuyentran1401.github.io/reproducible-data-science/structure_project/introduction.html

**Comment by pjbull** [comment]
Released in v2.3.0. Upgrade and use the latest version to check it out: https://pypi.org/project/cookiecutter-data-science/
